pkg/scheduler/internal/parallelize/parallelism.go

```diff
@@ -39,5 +41,5 @@ func chunkSizeFor(n int) workqueue.Options {
 
 // Until is a wrapper around workqueue.ParallelizeUntil to use in scheduling algorithms.
 func Until(ctx context.Context, pieces int, doWorkPiece workqueue.DoWorkPieceFunc) {
```

Comment:
better pass it as a parameter

Reply:
Do you mean passing `Parallelism` as an argument?

Reply:
yes

Reply:
still looks outstanding

Reply:
this is answered in the other thread